soccer/gameplay/tactics/stopped_plays/circle_near_ball.py

```diff
@@ -0,0 +1,147 @@
+                new_angle = (i - circle_ball.center()).angle()
+                new_angle = self.normalize_angle(new_angle)
+                angles.append(new_angle)
+
+            # Get angles going sequencially
+            angles.sort()
+
+            counter = 1
+            while counter < len(angles):
+                candidate_arcs.append(robocup.Arc(circle_ball.center(), radius, angles[counter - 1], angles[counter]))
+                counter = counter + 1
+            candidate_arcs.append(robocup.Arc(circle_ball.center(), radius, angles[len(angles) - 1], angles[0]))
+
+            i = 0
+            while i < len(candidate_arcs):
+                angle_between = candidate_arcs[i].end() - candidate_arcs[i].start()
+                angle_between = self.normalize_angle(angle_between)
+
+                angle_diff = candidate_arcs[i].start() + (angle_between) / 2.0
+                angle_diff = self.normalize_angle(angle_diff)
+
+                midpoint = (candidate_arcs[i].center() + robocup.Point(radius, 0))
+                midpoint.rotate(candidate_arcs[i].center(), angle_diff)
+                if not constants.Field.FieldRect.contains_point(midpoint):
+                    candidate_arcs.pop(i)
+                else:
+                    i = i + 1
+
+            candidate_arcs.sort(key=lambda arc: self.normalize_angle(arc.end() - arc.start()), reverse=True)
+
+            if len(candidate_arcs) <= 0:
+                final_arc = robocup.Arc(CircleNearBall.BackupBallLocation, radius, math.pi / 2, 5 * math.pi / 2)
+            else:
+                final_arc = candidate_arcs[0]
+        else:
+            midpoint = (circle_ball.center() + robocup.Point(radius, 0))
+            if not constants.Field.FieldRect.contains_point(midpoint):
+                final_arc = robocup.Arc(CircleNearBall.BackupBallLocation, radius, math.pi / 2, 5 * math.pi / 2)
+            else:
+                final_arc = robocup.Arc(circle_ball.center(), radius, math.pi / 2, 5 * math.pi / 2)
+
+        arc_angle = final_arc.end() - final_arc.start()
+        arc_angle = self.normalize_angle(arc_angle)
+
+        perRobot = arc_angle / (num_robots + 1)
+
+        dirvec = robocup.Point(radius, 0)
+        dirvec.rotate(robocup.Point(0,0), final_arc.start())
+        dirvec.rotate(robocup.Point(0,0), perRobot)
+
+        for i in range(6):
+            pt = final_arc.center() + dirvec
+            self.subbehavior_with_name("robot" + str(i)).pos = pt
+            dirvec.rotate(robocup.Point(0,0), perRobot)
+
+        # set robot attributes
+        for b in self.all_subbehaviors():
+            if b.robot is not None:
+                b.robot.set_avoid_ball_radius(constants.Field.CenterRadius)
+                b.robot.face(main.ball().pos)
+                b.robot.avoid_all_teammates(True)
+
+    def execute_completed(self):
+        self.move_around_circle()
+
+    # Makes an angle > 0, < pi * 2
+    def normalize_angle(self, angle):
```

Comment:
We have `robocup.fix_angle_radians` which does this same thing, so let's use that instead.